polaris-react/src/components/Avatar/Avatar.scss

```diff
@@ -81,21 +79,61 @@
 .sizeExtraSmall {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   width: var(--pc-avatar-extra-small-size);
+
+  #{$se23} &.shapeSquare {
+    // stylelint-disable-next-line -- se23 overrides
+    border-radius: 4px;
+  }
 }
 
 .sizeSmall {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   width: var(--pc-avatar-small-size);
+
+  #{$se23} &.shapeSquare {
+    // stylelint-disable-next-line -- se23 overrides
+    border-radius: 6px;
```

Comment:
@aaronccasanova Did you hear anything back about how our border-radius scales might be changing moving forward? Also curious about this since I have a 3px one [in Timeline](https://github.com/Shopify/web/pull/94898#discussion_r1240404604).

Reply:
Thanks for the callout @lgriffee 👍 @sarahill Do you have any insight on this? Note: I intentionally didn't use Polaris tokens for all square `Avatar` border radii to move quickly and wasn't sure if we wanted `border-radius-*` scale changes to affect what looked like custom values per size. We could add experimental component-specific `border-radius` tokens for `Avatar` to avoid unintended changes. However, I'm also trying to minimize that pattern since the team has generally discouraged it in the past.

Reply:
No haven't heard about that. It feels like it should be ok to merge this and then we can adjust as needed in a follow up PR?

Reply:
I would also prefer to follow up if needed 👍